### DISCUSSION TITLE
brighteness changes when switching from 1080p to other resolutions on a specific video

### DISCUSSION THREAD

**Discussion by ghost** [libre-tube/LibreTube#6169] [General]
### Steps to reproduce

1. Open https://youtu.be/DMv9XMR24lg
2. Play and pause it
3. Switch resolution to 1080
4. Switch resolution to a lower resolution (eg 480)
5. Redo switching

### Expected behavior

Brightness level will remain the same

### Actual behavior

After changing resolution to a value lower than 1080, the brightness changes.

If you have "night light" on, the brightness changes are also visible in LibreTube UI.

### LibreTube version

0.23.2

### Android version

13 (eOS 2.1)

### Other details

Doing the same from youtube web does not show any brightness change

Device: samsung a52q

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
I was kind of able to reproduce the issue.

I noticed that the video has been playing very laggy generally on higher resolutions, so I think there's some bug in the ExoPlayer library that causes issues decoding certain streams.

This is certainly not LibreTube's fault though, we don't do any video rendering/decoding ourselves, it's all handled by the ExoPlayer library.
